src/test/java/uk/gov/companieshouse/presenter/account/controller/PresenterAccountControllerTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.presenter.account.controller;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Do you need this if you're not actually mocking anything?

Reply:
yes it is not required.